Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -384,4 +356,269 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
     }
 
     return lines;
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _DWriteGetFontFamilyNames() {
+    NSMutableArray<NSString*>* fontFamilyNames = [NSMutableArray<NSString*> array];
+
```

Comment:
If we're committed to that for the long term, I guess I'll bring back in the CF versions of the code I wrote.

---
In reply to: [78610729](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78610729) [](ancestors = 78610729,78225402,78221279,78220974)